### PR TITLE
Correct and standardize package names

### PR DESCRIPTION
**Pull request by NobleKangaroo**
Some of the package names are incorrect (NetworkManager, avahi) or unnecessarily upper-cased (AppArmor). Standardize all package names to ones available in Debian stable repositories.
```bash
root@hass01:~# for PACKAGE in systemd network-manager apparmor docker jq curl avahi-daemon dbus; do apt info "${PACKAGE}" 2> /dev/null | grep "^Package:"; done    
Package: systemd
Package: network-manager
Package: apparmor
Package: docker
Package: jq
Package: curl
Package: avahi-daemon
Package: dbus
```

### CODE REVIEW

**Comment by NobleKangaroo** [comment]
Accidentally allowed #ebd6fdf to bleed over into this PR; it's been fixed. This PR is just the package names now.

**Review by pvizeli — APPROVED** [review]

**Comment by pvizeli** [comment]
You need rebease it

File: installer.sh [hunk 1]
```diff
@@ -47,14 +47,14 @@ URL_SERVICE_HASSIO="${URL_RAW_BASE}/hassio-supervisor.service"
...
 # Check env
-command -v systemctl > /dev/null 2>&1 || MISSING_PACKAGES+=("systemd")
-command -v nmcli > /dev/null 2>&1 || MISSING_PACKAGES+=("NetworkManager")
-command -v apparmor_parser > /dev/null 2>&1 || MISSING_PACKAGES+=("AppArmor")
-command -v docker > /dev/null 2>&1 || MISSING_PACKAGES+=("docker")
-command -v jq > /dev/null 2>&1 || MISSING_PACKAGES+=("jq")
-command -v curl > /dev/null 2>&1 || MISSING_PACKAGES+=("curl")
-command -v avahi-daemon > /dev/null 2>&1 || MISSING_PACKAGES+=("avahi")
-command -v dbus-daemon > /dev/null 2>&1 || MISSING_PACKAGES+=("dbus")
+command -v systemctl > /dev/null 2>&1 || MISSING_PACAKGES+=("systemd")
```
Comment (ludeeus) [inline installer.sh:50]: Your `MISSING_PACAKGES` should be  `MISSING_PACKAGES`

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: installer.sh (hunk 1 above)
Comment (NobleKangaroo) [reply installer.sh:50]: > Your `MISSING_PACAKGES` should be `MISSING_PACKAGES`

Haha thanks! Good eyes! 👁️👁️ I guess I had made my feature branch before the last PR to fix the typos was approved and merged. Fixing it up now!

**Comment by NobleKangaroo** [comment]
@ludeeus Resolved conflicts. Thanks!

**Pull request merged**